research-software-directory/RSD-as-a-service
# Minor UI fixes

# Two minor UI fixes 

Changes proposed in this pull request:

*  Project card to show image on small screens. This feature was lost during the last refactoring, we reverted to the version from previous release (v1.11.1)
*  Project card layout on mobile is further improved, the card orientation is changed to portrait for mobile while on larger screens remains to be landscape
*  Contact card layout is improved concerning the margins and the role of contact person is added
*  Software card and organisation card have similar approach for small screens (mobile, laptop portrait, laptop landcape)
*  Page title is not fixed when device height or width is lower than 640px

How to test:
* `make start` to rebuild everything and create example entries
* navigate to projects overview, the images should be visible even when you switch to mobile view
* contact person card will show the role if the role is specified

## Example project card in portrait for mobile
![image](https://user-images.githubusercontent.com/9204081/204842641-c9939dde-441d-4362-93cf-a85c91d06855.png)

## Example project card in landscape for tablet
![image](https://user-images.githubusercontent.com/9204081/204928101-7ef6650c-db63-4c24-94dd-f85b4a99667f.png)

## Example project card for desktop
![image](https://user-images.githubusercontent.com/9204081/204928253-26934504-7d0e-47a5-88fc-5626d6bc2a3c.png)

## Example organisation card in portrait for mobile
![image](https://user-images.githubusercontent.com/9204081/204927513-5ba007ae-d7bf-41dd-83c1-750750d25bea.png)

## Example organisation card for tablet
![image](https://user-images.githubusercontent.com/9204081/204927684-c779e307-d143-4f19-9ae2-92cc47176627.png)

## Example organisation card in portrait for desktop
![image](https://user-images.githubusercontent.com/9204081/204927857-8f1472aa-57eb-4871-b7c8-1eb5b38bb512.png)

## Example contact person card
![image](https://user-images.githubusercontent.com/9204081/204836942-1546356a-e47e-4568-ad66-8ff8ea01df91.png)

PR Checklist:
